website/style.css

```diff
@@ -56,7 +56,7 @@ select{
 
 input[type=text] {
   background-color: #eee;
-  width: 2.3em;
+  width: 3.5em;
```

Comment:
In my browser, 3.5 em is not enough to show 6 characters in caps. 4.5em does the trick for me. Please check.

Reply:
Solved with https://github.com/phl0/firmware/commit/9bece9392f208d0897e485a552e37f3b87ffc408. (Will rebase to 1.1.0 dev branch as soon as all issues are done).